lib/commands.js

```diff
@@ -846,16 +849,18 @@ function stackName(name, suffix) {
  * @param {object} oldParameters - name/value pairs defining old or default parameters
  * @param {object} newParameters - name/value pairs defining the new, unchanged old, or accepted default parameters
  * @param {object} [overrideParameters={}] - name/value pairs for any parameter values passed as overrides
+ * @param {boolean} isCreate - indicates that UsePreviousValue shoudld not be used on stack create. set in createPreable().
  * @returns {array} params - parameters objects for use in ChangeSet requests that create/update a stack
  */
-function changesetParameters(oldParameters, newParameters, overrideParameters) {
+function changesetParameters(oldParameters, newParameters, overrideParameters, isCreate) {
   overrideParameters = overrideParameters || {};
 
   return Object.entries(newParameters).map(([key, value]) => {
     const parameter = {
       ParameterKey: key
     };
-    if (!overrideParameters[key] && oldParameters[key] === newParameters[key]) {
+
+    if (!isCreate && !overrideParameters[key] && oldParameters[key] === newParameters[key]) {
```

Comment:
It would lend some clarity if we were to break up these conditionals. and try to avoid so many `!` Maybe something like this?

```suggestion
    const unchanged = oldParameters[key] === newParameters[key];
    const isOverriden = overrideParameters[key] && unchanged;
    
    if (isCreate || isOverriden) parameter.ParameterValue = value;
    else if (unchanged) parameter.UsePreviousValue = true;
    else parameter.ParameterValue = value;
```